src/tools/admin/conseillers/change-date-naissance.js

```diff
@@ -0,0 +1,46 @@
+const { program } = require('commander');
+const { execute } = require('../../utils');
+const dayjs = require('dayjs');
+
+const formatDate = date => {
+  return dayjs(date, 'YYYY-MM-DD').toDate();
+};
+
+execute(__filename, async ({ db, logger, Sentry, exit }) => {
+
+  program.option('-d, --date <type>', 'date : date de naissance sous le format AAAA/MM/DD');
+  program.option('-i, --id <id>', 'id: id PG du conseiller');
+  program.helpOption('-e', 'HELP command');
+  program.parse(process.argv);
+
+  const id = Math.floor(program.id);
```

Comment:
Je n'avais pas compris que c'était pour passer de string à int, j'imagine que c'est plutôt ça que tu voulais faire à la base :
```suggestion
 const id = +program.id;
```
Après je pense que ce qui revèle le mieux ton intention c'est :
```suggestion
 const id = Number(program.id);
```